config/environments/development.rb

```diff
@@ -84,6 +84,8 @@
 ENV['ADMIN_PASSWORD'] = 'password'
 
 # General use
-ENV['DAY_TO_SEND_INVITES'] = 'wednesday'
+ENV['DAY_TO_SEND_INVITES'] = 'friday'
+# DAY_TO_SEND_RESULTS_EMAIL must be sent on the day when time_period.end_date <= Date.current
```

Comment:
For things that need to be done at a later date please have as a `TODO: ...` comment